app/mailers/rad_mailer.rb

```diff
@@ -114,4 +115,43 @@ def escape_name(recipient_name)
     def enable_settings_link
       @notification_settings_link = true
     end
+
+    def create_contact_log
+      ContactLog.create!(
+        from_email: mail.from.first,
+        reply_to: mail.reply_to.first,
+        subject: mail.subject,
+        message: @message, # What to store here if anything? could be html template
+        log_type: :outgoing,
+        service_type: :sendgrid
+      ).tap do |log|
+        create_contact_log_attachments(log) # Do we want to store all of these?
```

Comment:
probably not, seems like a lot - why do we have this attachments model to begin with?

Reply:
let's ask @elbzero if he knows, maybe we can get rid of it but if we need to keep it, and we don't store the attachments on emails but do store them on SMS then we might want to rename the table/model so it's not confusing